scripts/linux/runE2ETest.sh

```diff
@@ -22,6 +22,7 @@ function clean_up() {
     if [ "$CLEAN_ALL" = '1' ]; then
         echo 'Prune docker system'
         docker system prune -af --volumes || true
+        docker network prune -f || true
```

Comment:
you don't need to call "docker network prune" as "docker system prune" will do.  The reason why it doesn't work now coz we need to remove all the containers first before calling docker system prune.  I have a PR out to fix it.